docs/resources/project_badge.md

```diff
@@ -0,0 +1,39 @@
+# gitlab\_project\_badge
```

Comment:
Can you remove the consecutive newlines (line 7 and 32), and add a terminating newline character to this file?

We very recently added the [markdown-linter](https://github.com/avto-dev/markdown-lint) to our CI, and it will complain about these after we merge to master.

Reply:
@armsnyder Done. I tested it with the same tool and it's ok now.

Reply:
Thanks for checking